subgraphs/compound-forks/protocols/aurigami/src/mapping.ts

```diff
@@ -333,6 +350,184 @@ function getOrCreateProtocol(): LendingProtocol {
 //// Helpers ////
 /////////////////
 
+// calculate PLY reward speeds
+function updateRewards(event: ethereum.Event, marketID: Address): void {
+  const protocol = getOrCreateProtocol();
+  const market = Market.load(marketID.toHexString());
+  if (!market) {
+    log.warning("Market not found for address {}", [marketID.toHexString()]);
+    return;
+  }
+  const auriLensContract = AuriLens.bind(AURI_LENS_CONTRACT_ADDRESS);
+  const tryRewardSpeeds = auriLensContract.try_getRewardSpeeds(
+    comptrollerAddr,
+    event.address
+  );
+
+  if (tryRewardSpeeds.reverted) {
+    log.warning("Could not get borrow/supply speeds for market {}", [
+      market.id,
+    ]);
+    return;
+  }
+
+  const rewardTokens: string[] = [];
+  const rewardsAmount: BigInt[] = [];
+  const rewardsAmountUSD: BigDecimal[] = [];
+
+  // PLY Borrow first since Reward tokens will be alphabetized
+  // Ply Borrow, Ply Supply, AURORA Borrow, AURORA Supply
```

Comment:
Shouldn’t Aurora go before PLY then?

Reply:
based off of the token address not the name

Reply:
Hmm, but the `rewardToken.id` includes the reward type. So then it should be PLY Borrow, Aurora Borrow, PLY Supply, Aurora Supply?